lib/metasploit/framework/login_scanner/base.rb

```diff
@@ -45,6 +45,20 @@ module Base
           # @!attribute bruteforce_speed
           #   @return [Integer] The desired speed, with 5 being 'fast' and 0 being 'slow.'
           attr_accessor :bruteforce_speed
+          # @!attribute maxconsecutiveerrors
+          #   @return [Integer] Maximum consecutive errors allowed
+          attr_accessor :maxconsecutiveerrors
```

Comment:
We'll want to use underscores for this, including the documentation plus other introduced variables

```suggestion
          attr_accessor :max_consecutive_error_count
```